js/main.js

```diff
@@ -37,13 +38,13 @@ Mousetrap.bind(['command+c', 'ctrl+c'], function() {
 			selectedCubelet = activeNode.cubelet;
 		}
 	}
-	reGenerate();
+	regenerate();
 	return false;
 });
 
 //shortcut to add a node
-Mousetrap.bind(['command+a', 'ctrl+a', 'plus'], function() {
```

Comment:
I removed `mod+A`, `mod+D`, and `mod+C` because two of them override platform behaviour: _Select All_, and _Copy_. It made it difficult to copy/paste stuff out of our UI when reporting issues.

The `+`/`-` keys still work, on a numpad or otherwise, and that's what we're expecting our users to use.